### PR TITLE
Ndmg bids to python3

### PR DESCRIPTION
**Pull request by loftusa**
Essentially just a few edits to setup.py and a python version check in ndmg_bids. Everything else is just black formatting

### CODE REVIEW

File: ndmg/scripts/ndmg_cloud.py [hunk 1]
```diff
@@ -56,7 +55,7 @@ def batch_submit(
     log=False,
     bg=False,
     modif="",
-    reg_style=""
+    reg_style="",
```
Comment (dPys) [inline ndmg/scripts/ndmg_cloud.py:58]: Why is there a comma here now?

File: ndmg/scripts/ndmg_cloud.py [hunk 2]
```diff
@@ -136,29 +145,35 @@ def crawl_bucket(bucket, path, jobdir):
...
+    bucket,
+    path,
+    threads,
+    jobdir,
+    credentials=None,
+    debug=False,
+    dataset=None,
+    bg=False,
+    modif="",
+    reg_style="",
```
Comment (dPys) [inline ndmg/scripts/ndmg_cloud.py:162]: Same here, why is there a comma after reg_style="",
?

**Review by dPys — CHANGES_REQUESTED** [review]
Check for rogue commas

File: ndmg/scripts/ndmg_cloud.py (hunk 1 above)
Comment (loftusa) [reply ndmg/scripts/ndmg_cloud.py:58]: I don't think I added that myself, so it might be a Black thing. Entirely stylistic

Comment (dPys) [reply ndmg/scripts/ndmg_cloud.py:58]: Please confirm that it doesn't break anything

**Review by dPys — APPROVED** [review]

**Pull request merged**